simple/sample/input/config.json

```diff
@@ -19,5 +19,23 @@
     "wikidataids.csv": {
       "entityType": "Country"
     }
+  },
+  "variables": {
+    "var1": {
+      "name": "Good var1 name",
+      "description": "Good var1 description",
+      "nlSentences": [
+        "Natural language sentence 1",
+        "Natural language sentence 2"
+      ],
```

Comment:
I still think separate out the nl descriptions into a separate csv file is cleaner.

Reply:
Discussed offline to keep it here for now. Will consider moving to a different file in the future if it's helpful.